testhelpers/docker_registry.go

```diff
@@ -22,13 +32,23 @@ var registryImageNames = map[string]string{
 }
 
 func NewDockerRegistry() *DockerRegistry {
-	return &DockerRegistry{}
+	return &DockerRegistry{
+		Name: "test-registry-" + RandString(10),
+	}
+}
+
+func NewDockerRegistryWithAuth(dockerConfigDir string) *DockerRegistry {
+	return &DockerRegistry{
+		Name:            "test-registry-" + RandString(10),
+		username:        RandString(10),
+		password:        RandString(10),
+		DockerDirectory: dockerConfigDir,
+	}
 }
 
-func (registry *DockerRegistry) Start(t *testing.T) {
-	t.Log("run registry")
+func (r *DockerRegistry) Start(t *testing.T) {
+	t.Log("run r")
```

Comment:
```suggestion
	t.Log("run registry")
```
Search and replace? :)

Reply:
Good catch!